yaksh/views.py

```diff
@@ -176,9 +185,24 @@ def edit_question(request, question_id=None):
         raise Http404('No Question Found')
 
     question_instance = Question.objects.get(id=question_id)
-
+    if request.method == "POST" and 'delete_files' in request.POST:
+        remove_files_id = request.POST.getlist('clear')
+        if remove_files_id:
+            files = QuestionsFileUpload.objects.filter(id__in=remove_files_id)
+            file = QuestionsFileUpload()
+            file.delete_files(files)
     if request.method == "POST" and 'save_question' in request.POST:
         question_form = QuestionForm(request.POST, instance=question_instance)
+        form = FileForm(request.POST, request.FILES)
+        files = request.FILES.getlist('file_field')
+        extract_files_id = request.POST.getlist('extract')
+        if files:
+            for file in files:
+                QuestionsFileUpload.objects.get_or_create(question=question_instance, files=file)
+        if extract_files_id:
+            files = QuestionsFileUpload.objects.filter(id__in=extract_files_id)
+            file = QuestionsFileUpload()
+            file.set_extract_status(files)
```

Comment:
Same issue.
If we want a custom model method then it should accessible in following manner:
Example:
course = Course.objects.get(pk=1)
course.deactivate()
deactivate is model method.